lib/transforms/json-to-html-data.js

```diff
@@ -212,15 +219,13 @@ function getMaterials (data) {
 }
 
 function getMeasurements (data) {
-  const value =
-    getNestedProperty(data, 'attributes.component.0.measurements.display') ||
-    getNestedProperty(data, 'attributes.measurements.dimensions.0.value');
-  return value ? {key: 'Measurements', value: value} : null;
-}
-
-function getStatus (data) {
-  const value = getNestedProperty(data, 'attributes.legal.legal_status');
-  return value ? {key: 'Status', value: value} : null;
+  const value = [];
```

Comment:
if I was being picky, I'd suggest calling this variable `values` and not making it a `const` given that it's being pushed to...